### DISCUSSION TITLE
results on real sences

### DISCUSSION THREAD

**Discussion by cv-lab-x** [NVlabs/instant-ngp#398] [Q&A]
Hi, thanks for your great work. i have a little question about the results on real sences. Did you test the network on some real sences for example DTU or Tanks and Temples datasets?
looking forward to your reply, thanks a lot!

**Comment by Tom94** [comment]
Hi! Yes, the codebase works with real scenes that come in the standard NeRF `.json` format. The bundled `fox` dataset as well as the videos of the robot and modular synthesizer come from self-captured real-world scenes in that format.
